LayoutTests/svg/hittest/resources/svg-tooltip.js

```diff
@@ -11,7 +11,7 @@
 // Returns true if an element can return a title or not
 function canReturnTitle(element)
 {
-  if (!element || element.tagName == "svg" && element.ownerDocument instanceof SVGDocument)
+  if (!element || (element.tagName === "svg" && element.ownerDocument instanceof Document && element.namespaceURI === "http://www.w3.org/2000/svg"))
```

Comment:
It might be interesting to figure out why there was a need for `&& element.ownerDocument instanceof SVGDocument` in the first place in the test.

The test was added in https://bugs.webkit.org/show_bug.cgi?id=139690 from 2014.

https://github.com/WebKit/WebKit/blob/a227ef91c9162e0519ccc17f99184bde84eac46e/LayoutTests/svg/hittest/resources/svg-tooltip.js#L12-L18

The test says:

```
// This code mimics the C++ code for calculating the tooltip text for an SVG element.
```

The current code to return the title inside SVG in the c++ code is 

https://searchfox.org/wubkat/rev/72adbb9f99534551a728acb270d4f111da5e9515/Source/WebCore/svg/SVGElement.cpp#820-829 

```cpp
String SVGElement::title() const
{
    // According to spec, for stand-alone SVG documents we should not return a title when
    // hovering over the rootmost SVG element (the first <title> element is the title of
    // the document, not a tooltip) so we instantly return.
    if (isOutermostSVGSVGElement() && document().topDocument().isSVGDocument())
        return String();
    RefPtr firstTitle = childrenOfType<SVGTitleElement>(*this).first();
    return firstTitle ? const_cast<SVGTitleElement&>(*firstTitle).innerText() : String();
}
```

So it tests if it is the `svg` root element AND if it is a standalone document (aka not inside an HTML document).

Another way to test this would be to test the `element.parentElement` It will be null in a standalone SVG for the `svg` element, while it would return another element with SVG embedded inside HTML.